app/src/main/java/com/keylesspalace/tusky/fragment/PreferencesFragment.java

```diff
@@ -58,23 +60,28 @@ public void onCreate(Bundle savedInstanceState) {
 
         if(notificationPreferences != null) {
 
+            AccountEntity activeAccount = TuskyApplication.getAccountManager().getActiveAccount();
+
+            if (Build.VERSION.SDK_INT < Build.VERSION_CODES.O && activeAccount != null) {
+                notificationPreferences.setSummary(getString(R.string.pref_summary_notifications, activeAccount.getFullName()));
+            }
+
+
             //on Android O and newer, launch the system notification settings instead of the app settings
             if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
 
-                 NotificationManager.createNotificationChannels(getContext());
-
-                    notificationPreferences.setOnPreferenceClickListener(new Preference.OnPreferenceClickListener() {
-                        @Override
-                        public boolean onPreferenceClick(Preference preference) {
-                            Intent intent = new Intent();
-                            intent.setAction("android.settings.APP_NOTIFICATION_SETTINGS");
+                notificationPreferences.setOnPreferenceClickListener(new Preference.OnPreferenceClickListener() {
```

Comment:
Ugh, oh, it crashed for me
Could it be Lineage?

```
02-03 19:34:45.642 19362-19362/com.keylesspalace.tusky.test I/art: Rejecting re-init on previously-failed class java.lang.Class<android.support.v4.app.JobIntentService$JobServiceEngineImpl>: java.lang.NoClassDefFoundError: Failed resolution of: Landroid/app/job/JobServiceEngine;
02-03 19:34:45.642 19362-19362/com.keylesspalace.tusky.test I/art:     at void com.evernote.android.job.JobRescheduleService.startService(android.content.Context) (JobRescheduleService.java:29)
02-03 19:34:45.642 19362-19362/com.keylesspalace.tusky.test I/art:     at void com.evernote.android.job.JobManager.<init>(android.content.Context) (JobManager.java:159)
02-03 19:34:45.642 19362-19362/com.keylesspalace.tusky.test I/art:     at com.evernote.android.job.JobManager com.evernote.android.job.JobManager.create(android.content.Context) (JobManager.java:112)
02-03 19:34:45.642 19362-19362/com.keylesspalace.tusky.test I/art:     at void com.keylesspalace.tusky.TuskyApplication.onCreate() (TuskyApplication.java:70)
02-03 19:34:45.642 19362-19362/com.keylesspalace.tusky.test I/art:     at void android.app.Instrumentation.callApplicationOnCreate(android.app.Application) (Instrumentation.java:1025)
02-03 19:34:45.642 19362-19362/com.keylesspalace.tusky.test I/art:     at void android.app.ActivityThread.handleBindApplication(android.app.ActivityThread$AppBindData) (ActivityThread.java:5448)
02-03 19:34:45.642 19362-19362/com.keylesspalace.tusky.test I/art:     at void android.app.ActivityThread.-wrap2(android.app.ActivityThread, android.app.ActivityThread$AppBindData) (ActivityThread.java:-1)
02-03 19:34:45.642 19362-19362/com.keylesspalace.tusky.test I/art:     at void android.app.ActivityThread$H.handleMessage(android.os.Message) (ActivityThread.java:1564)
02-03 19:34:45.642 19362-19362/com.keylesspalace.tusky.test I/art:     at void android.os.Handler.dispatchMessage(android.os.Message) (Handler.java:102)
02-03 19:34:45.642 19362-19362/com.keylesspalace.tusky.test I/art:     at void android.os.Looper.loop() (Looper.java:154)
02-03 19:34:45.642 19362-19362/com.keylesspalace.tusky.test I/art:     at void android.app.ActivityThread.main(java.lang.String[]) (ActivityThread.java:6186)
02-03 19:34:45.642 19362-19362/com.keylesspalace.tusky.test I/art:     at java.lang.Object java.lang.reflect.Method.invoke!(java.lang.Object, java.lang.Object[]) (Method.java:-2)
02-03 19:34:45.642 19362-19362/com.keylesspalace.tusky.test I/art:     at void com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run() (ZygoteInit.java:889)
02-03 19:34:45.642 19362-19362/com.keylesspalace.tusky.test I/art:     at void com.android.internal.os.ZygoteInit.main(java.lang.String[]) (ZygoteInit.java:779)
02-03 19:34:45.642 19362-19362/com.keylesspalace.tusky.test I/art: Caused by: java.lang.ClassNotFoundException: Didn't find class "android.app.job.JobServiceEngine" on path: DexPathList[[zip file "/data/app/com.keylesspalace.tusky.test-2/base.apk", zip file "/data/app/com.keylesspalace.tusky.test-2/split_lib_dependencies_apk.apk", zip file "/data/app/com.keylesspalace.tusky.test-2/split_lib_slice_0_apk.apk", zip file "/data/app/com.keylesspalace.tusky.test-2/split_lib_slice_1_apk.apk", zip file "/data/app/com.keylesspalace.tusky.test-2/split_lib_slice_2_apk.apk", zip file "/data/app/com.keylesspalace.tusky.test-2/split_lib_slice_3_apk.apk", zip file "/data/app/com.keylesspalace.tusky.test-2/split_lib_slice_4_apk.apk", zip file "/data/app/com.keylesspalace.tusky.test-2/split_lib_slice_5_apk.apk", zip file "/data/app/com.keylesspalace.tusky.test-2/split_lib_slice_6_apk.apk", zip file "/data/app/com.keylesspalace.tusky.test-2/split_lib_slice_7_apk.apk", zip file "/data/app/com.keylesspalace.tusky.test-2/split_lib_slice_8_apk.apk", zip file "/data/app/com.keylesspalace.tusky.te
02-03 19:34:45.642 19362-19362/com.keylesspalace.tusky.test I/art:     at java.lang.Class dalvik.system.BaseDexClassLoader.findClass(java.lang.String) (BaseDexClassLoader.java:56)
02-03 19:34:45.642 19362-19362/com.keylesspalace.tusky.test I/art:     at java.lang.Class java.lang.ClassLoader.loadClass(java.lang.String, boolean) (ClassLoader.java:380)
02-03 19:34:45.642 19362-19362/com.keylesspalace.tusky.test I/art:     at java.lang.Class java.lang.ClassLoader.loadClass(java.lang.String) (ClassLoader.java:312)
02-03 19:34:45.642 19362-19362/com.keylesspalace.tusky.test I/art:     at void com.evernote.android.job.JobRescheduleService.startService(android.content.Context) (JobRescheduleService.java:29)
02-03 19:34:45.642 19362-19362/com.keylesspalace.tusky.test I/art:     at void com.evernote.android.job.JobManager.<init>(android.content.Context) (JobManager.java:159)
02-03 19:34:45.642 19362-19362/com.keylesspalace.tusky.test I/art:     at com.evernote.android.job.JobManager com.evernote.android.job.JobManager.create(android.content.Context) (JobManager.java:112)
02-03 19:34:45.642 19362-19362/com.keylesspalace.tusky.test I/art:     at void com.keylesspalace.tusky.TuskyApplication.onCreate() (TuskyApplication.java:70)
02-03 19:34:45.642 19362-19362/com.keylesspalace.tusky.test I/art:     at void android.app.Instrumentation.callApplicationOnCreate(android.app.Application) (Instrumentation.java:1025)
02-03 19:34:45.642 19362-19362/com.keylesspalace.tusky.test I/art:     at void android.app.ActivityThread.handleBindApplication(android.app.ActivityThread$AppBindData) (ActivityThread.java:5448)
02-03 19:34:45.642 19362-19362/com.keylesspalace.tusky.test I/art:     at void android.app.ActivityThread.-wrap2(android.app.ActivityThread, android.app.ActivityThread$AppBindData) (ActivityThread.java:-1)
02-03 19:34:45.642 19362-19362/com.keylesspalace.tusky.test I/art:     at void android.app.ActivityThread$H.handleMessage(android.os.Message) (ActivityThread.java:1564)
02-03 19:34:45.642 19362-19362/com.keylesspalace.tusky.test I/art:     at void android.os.Handler.dispatchMessage(android.os.Message) (Handler.java:102)
02-03 19:34:45.642 19362-19362/com.keylesspalace.tusky.test I/art:     at void android.os.Looper.loop() (Looper.java:154)
02-03 19:34:45.642 19362-19362/com.keylesspalace.tusky.test I/art:     at void android.app.ActivityThread.main(java.lang.String[]) (ActivityThread.java:6186)
02-03 19:34:45.642 19362-19362/com.keylesspalace.tusky.test I/art:     at java.lang.Object java.lang.reflect.Method.invoke!(java.lang.Object, java.lang.Object[]) (Method.java:-2)
02-03 19:34:45.642 19362-19362/com.keylesspalace.tusky.test I/art:     at void com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run() (ZygoteInit.java:889)
02-03 19:34:45.642 19362-19362/com.keylesspalace.tusky.test I/art:     at void com.android.internal.os.ZygoteInit.main(java.lang.String[]) (ZygoteInit.java:779)
```

Reply:
This one is very strange, because it seems to be a problem with the job scheduler library we use and Lineage. But then, why does Tusky 1.4 not crash for you?

Reply:
@connyduck maybe it did? I don't remember checking notification settings, to be honest